lib/database/cloud_database.dart

```diff
@@ -52,3 +54,36 @@ final migrateCloudV3ToV4 = Migration(3, 4, (database) async {
       ALTER TABLE Persona ADD COLUMN ethereumIndex INTEGER NOT NULL DEFAULT(1);
       """);
 });
+
+final migrateCloudV4ToV5 = Migration(4, 5, (database) async {
+  await database.execute("""
+      ALTER TABLE Persona ADD COLUMN tezosIndexes TEXT;
+      """);
+  await database.execute("""
+      ALTER TABLE Persona ADD COLUMN ethereumIndexes TEXT;
+      """);
+
+  await database.execute("""
+      UPDATE Persona SET tezosIndexes = 
+        (WITH RECURSIVE
+          cnt(x, y, id) AS (
+            SELECT 0, "", Persona.tezosIndex
+            UNION ALL
+            SELECT x + 1, y || "," || x || "", Persona.tezosIndex FROM cnt
+            LIMIT 100
+          )
+        SELECT y FROM cnt WHERE x = (SELECT id FROM cnt WHERE x = 0));
+      """);
+
+  await database.execute("""
```

Comment:
It's quite complicated and will be easy to lose the indexes when getting bugs during updating.
How about creating a new address table like this:
```
id: primaryKey,
persona_id: foreignKey -> Persona(id)
crypto_type,
index,
address,
```
Can you evaluate does it possible and easy to change from this design?

Reply:
Create new table doesn't help with migration, still have to find a way to insert to that table using old index.
and i don't think it's easy to lose indexes. The script may be complicated, but it's easy to test. Since its input is only one positive integer, you only have to test 3 case 0, 1, n.
About implementation, it'll always cost effort to create new table